app/src/components/HomePage/DownloadAndShareModals/PublishedView.tsx

```diff
@@ -46,16 +46,9 @@ export function PublishedView({
       addSuffix: true,
       locale: getLocale(lng),
     });
-  try {
-    console.log("process.env", JSON.stringify(process.env)); // TODO NINA
-  } catch (e) {
-  } finally {
-    console.log(
-      "process.env.NEXT_PUBLIC_HOST",
-      JSON.stringify(process.env.NEXT_PUBLIC_HOST),
-    ); // TODO NINA
-  }
-  const URL = `${process.env.NEXT_PUBLIC_HOST}/${lng}/public/${inventoryId}`;
+
+  const URL = `${window.location.protocol}//${window.location.host}/${lng}/public/${inventoryId}`;
```

Comment:
I think you need the port after a : after the host in case there is a port number other than 80 in use